packages/web-components/src/tabs/tabs.ts

```diff
@@ -0,0 +1,186 @@
+import { attr, css, ElementStyles } from '@microsoft/fast-element';
+import { FASTTabs, TabsOrientation } from '@microsoft/fast-foundation';
+import { Tab } from '../index.js';
+import { TabsAppearance, TabsSize } from './tabs.options.js';
+
+export const TabTokenNames = {
+  tabIndicatorOffset: '--tabIndicatorOffset',
+  tabIndicatorScale: '--tabIndicatorScale',
+} as const;
```

Comment:
@bearcat-msft just my own $0.02, these aren't really used broadly. Having a constant for it seems like overkill. Not going to block, but I'm not sure we want these exposed to this extent.

Reply:
@chrisdholt we could just assign variable using `$fastControler.addStyles`

Reply:
np, removed this.